### ISSUE TITLE
Header does not fold up after refresh.

### ISSUE THREAD

**Issue by AbdulRehmanNazar** [licaomeng/Android-PullToRefresh-SwipeMenuListView-Sample#2]
Thank you for your nice Component design. I am facing a issue on the Meizu MX4 and VIVO device when we pull and release the list then the header doesn't fold up. check screenshots. Please  give any solution about it. Its urgent 

![1 pic_hd](https://cloud.githubusercontent.com/assets/6792823/11233626/85b187ae-8de1-11e5-9921-dbfa3206af8c.jpg)

![2 pic_hd](https://cloud.githubusercontent.com/assets/6792823/11233629/8acbf422-8de1-11e5-8294-fd68033e5f60.jpg)

**Comment by licaomeng** [comment]
Hi, you can refer to `line 261` in `PullToRefreshSwipeMenuListView.java`, `resetHeaderHeight()`  can realize the header reset.

**Comment by AbdulRehmanNazar** [comment]
Yes I checked that one and its works on some devices but it show issue on VIVO device and on Meizu device we have disable the OverScrollView and issue resolved.
But it still show issue on the VIVO device.

**Comment by AbdulRehmanNazar** [comment]
Here is the video link, please checkout this
http://1drv.ms/1Hc2Vi4
